src/net/sourceforge/kolmafia/persistence/CoinmastersDatabase.java

```diff
@@ -199,12 +202,22 @@ public static final Map<Integer, Integer> invert(final Map<Integer, Integer> map
 
           Map<Integer, Integer> map = getOrMakeMap(master, buyPrices);
           map.put(iitemId, iprice);
+
+          if (row != null) {
+            ShopRowDatabase.registerShopRow(row, "coin", item, master);
+          }
         } else if (type.equals("sell")) {
           List<AdventureResult> list = getOrMakeList(master, sellItems);
           list.add(item);
 
           Map<Integer, Integer> map = getOrMakeMap(master, sellPrices);
           map.put(iitemId, iprice);
+
+          if (row != null) {
+            System.out.println("sell row = " + row + "  for " + master);
```

Comment:
Will this print be removed eventually, or wrapped by something?  Not blocking :-)

Reply:
Good point. It was a DEBUG print.
I'll remove it now and merger it without further review. :)